core/src/test/java/org/elasticsearch/search/aggregations/bucket/DateRangeIT.java

```diff
@@ -286,17 +287,22 @@ public void testSingleValueFieldWithStringDatesWithCustomFormat() throws Excepti
     }
 
     public void testSingleValueFieldWithDateMath() throws Exception {
+        int timeZoneOffset = randomIntBetween(-12, 12);
```

Comment:
I just found out you can go nuts and even have time zones with arbitrary minute offset like `-03.14` which might be the negative pi time zone, but as much as I like this (and Nepal) I'd prefer to leave it like this. I think those odd offsets don't add to the thing tested here and make the test harder to read.